src/styles/index.scss

```
@@ -1,3 +1,132 @@
+$almost-white: #f3f3f3;
+$bg-white: #fff;
+$light-blue: #00acdc;
```

It is better to make the variables more universal.

Example:
```suggestion
$page-color: #f3f3f3;
$main-color: #fff;
$second-color: #00acdc;
```